src/scripts/components/colour/Time.js

```diff
@@ -5,10 +5,12 @@ import { TimeHelper } from '../../modules/timehelper';
 export const Time = (props) => {
   const time = props.time;
 
-  var hour = time.hour;
+  var hour = parseInt(time.hour);
```

Comment:
I was under the impression that `time.hour` is a `Number` though. I could be wrong -- I'd have to check later.